src/webhooks/registry.ts

```diff
@@ -143,31 +144,42 @@ function topicForStorage(topic: string): string {
   return topic.toUpperCase().replace(/\//g, '_');
 }
 
-export const webhookRegistry: {[topic: string]: WebhookHandlerFunction} = {};
-
-export function resetWebhookRegistry() {
-  for (const key in webhookRegistry) {
-    if ({}.hasOwnProperty.call(webhookRegistry, key)) {
-      delete webhookRegistry[key];
+export function resetHttpWebhookRegistry(
```

Comment:
We can probably get rid of this now that the state is local to the object (it's re-created on every test).